mllib/src/main/scala/org/apache/spark/ml/param/shared/SharedParamsCodeGen.scala

```diff
@@ -153,6 +154,11 @@ private[shared] object SharedParamsCodeGen {
     } else {
       ""
     }
+    val groupParam = if (param.isExpertParam) {
```

Comment:
Thanks, :) , but its next line [methodStr ](https://github.com/apache/spark/pull/14738/files#diff-400ce1c002287d1afbe8b8486a74ecddL156) also needs to be cleaned?